docs/user-guide/installation.rst

```diff
@@ -149,13 +149,13 @@ Tools like `virtualenv <https://virtualenv.pypa.io/en/latest/>`_ and `virtualenv
 
```

Comment:
@TyDunn can we just remove the tabs (L 142, 144) since all the instructions are the same now?

Reply:
@erohmensing I thought about it. I opted to keep the tab because it kept the design consistent across all steps with a nice border and explicit identification of the OS.

Reply:
Fair, the border makes sense